Client/Frontend/Strings.swift

```diff
@@ -1652,14 +1690,14 @@ extension String {
                 value: "Quick-Search Engines",
                 comment: "Title for quick-search engines settings section in the Search page in the Settings menu.")
             public static let PrivateSessionTitle = MZLocalizedString(
-                key: "Settings.Search.PrivateSession.Title.v121",
+                key: "Settings.Search.PrivateSession.Title.v122",
                 tableName: "Settings",
-                value: "Private Session",
+                value: "Private Browsing",
                 comment: "Title for the `Private Session` settings section in the Search page in the Settings menu.")
             public static let PrivateSessionSetting = MZLocalizedString(
                 key: "Settings.Search.PrivateSession.Setting.v121",
                 tableName: "Settings",
-                value: "Turn off suggestions in private browsing",
+                value: "Show Suggestions in Private Browsing",
```

Comment:
This string should get its key updated as well, otherwise I think the new value won't be exported.

Reply:
danm it. I missed this one.